Slim/Control/Queries.pm

```diff
@@ -792,9 +794,31 @@ sub albumsQuery {
 			$request->addResultLoopIfValueDefined($loopname, $chunkCount, 'work_id', $c->{'tracks.work'});
 			$request->addResultLoopIfValueDefined($loopname, $chunkCount, 'work_name', $c->{'works.title'});
 			$request->addResultLoopIfValueDefined($loopname, $chunkCount, 'composer', $c->{'composer.name'});
-			$request->addResultLoopIfValueDefined($loopname, $chunkCount, 'grouping', $c->{'tracks.grouping'});
+			$request->addResultLoop($loopname, $chunkCount, 'grouping', $c->{'tracks.grouping'}||"");
```

Comment:
It's because I need to have `grouping` as a `commonVariable` in `sub _albums` in order to split the list into work instances in the case when an album contains multiple performances of the same work, but it is often (maybe mostly) null, which makes Squeezer (slimbrowse clients generally?) fall over. 

This forces a value so that there is always a key:value pair passed.

If there is another way of doing this, please advise.